app/experimenter/nimbus-ui/.rescriptsrc.js

```diff
@@ -28,5 +29,26 @@ module.exports = [(config) => {
     filename: 'static/css/[name].css',
   });
 
+  if (process.env.SENTRY_UPLOAD_SOURCEMAPS && process.env.SENTRY_AUTH_TOKEN) {
+    let versionInfo = null;
+    try {
+      versionInfo = require("../../version.json");
+    } catch (e) {
+      /* no-op */
+    }
+    if (versionInfo) {
+      config.plugins.push(
+        new SentryWebpackPlugin({
+          authToken: process.env.SENTRY_AUTH_TOKEN,
+          org: process.env.SENTRY_ORG,
+          project: process.env.SENTRY_PROJECT,
+          release: versionInfo.commit,
+          include: "./build",
+          urlPrefix: "~/static/nimbus",
+        })
+      );
+    }
+  }
+
```

Comment:
This code should end up doing nothing until we manage to get these env vars set in Jenkins (or CircleCI?) for build:

* `SENTRY_ORG=operations`
* `SENTRY_PROJECT=nimbus-ui-prod` / `SENTRY_PROJECT=nimbus-ui-stage` (respectively)
* `SENTRY_AUTH_TOKEN=XXXX`
  * this value will need to be an auth token from [a new internal integration created within Developer Settings found under Sentry organizational settings](https://docs.sentry.io/product/integrations/integration-platform/#internal-integrations)